README.md

```diff
@@ -52,10 +54,17 @@ SCSS and Typescript.
 
 ## Credits
 
-Developed by <author-name> for <company-name>.
+Based on Piano Genie, created by Chris Donahue, Ian Simon, Sander Dieleman and part of the 
+[Google Magenta project](https://magenta.tensorflow.org/).
+
+Developed by Christian Stussak for IMAGINARY gGmbH.
 
 ## License
 
-Copyright © <year> <copyright-holder>
+Piano Genie:
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Modifications:
+Copyright 2023 IMAGINARY gGmbH.
 
-Licensed under the MIT license (see the [`LICENSE`](LICENSE) file).
+Licensed under the Apache License (see LICENSE).
```

Comment:
Technically, we would not need to change the license. Apache licensed code can be used from non-Apache licensed code. I am OK with changing it to Apache, though.

BTW: The license should actually be "Apache v2.0", not "Apache", which is what Google Magenta uses.